package.json

```diff
@@ -109,6 +109,7 @@
     "react-dropzone": "^4.2.1",
     "react-motion": "^0.5.2",
     "react-onclickoutside": "^6.6.2",
+    "react-paginate": "^5.2.3",
```

Comment:
You didn't commit the `yarn.lock` file for this change.

Reply:
Oh, my bad. I was under the impression that I don't have to commit `yarn.lock` file  :sweat_smile: